crates/hulk_replayer/src/labels.rs

```diff
@@ -52,6 +51,14 @@ impl<'state> Widget for Labels<'state> {
             {
                 child_ui.label(format!("{} frames", label_content.number_of_frames));
             }
+            if row_height
+                >= (3.0 * ui.style().text_styles.get(&TextStyle::Body).unwrap().size)
+                    + ui.spacing().item_spacing.y
```

Comment:
One spacing unit is missing, right?

Reply:
I realized both of the if conditions weren't quite right.
Now I'm using `Ui::available_height` to check if another label would fit into the remaining available space.